docs/installation/database/passing-configuration.rst

```diff
@@ -0,0 +1,430 @@
+
+.. py:currentmodule:: datacube
+
+ODC Configuration (Details)
+***************************
+
+Open Data Cube configuration files, supported configuration options, and basic use cases and defaults
+are described in :doc:`configuration`.
+
+The Open Data Cube uses configuration files and/or environment variables to determine how to connect to databases.
+
+Overview
+========
+
+The normal flow of configuration processing can be bypassed by `explicitly passing`_ in a complete configuration.
+Such explicit configuration blocks reading of configuration from environment variables.
```

Comment:
Is this sentence missing some words, or does it mean "Explicitly passing in a complete configuration disables the mechanism of reading environment variables for configuration"?

Reply:
Yes it means that, and yes, it just says that twice.  ....  Thanks I'll reword.